ocs_ci/ocs/ui_selenium/ui_operator.py

```diff
@@ -0,0 +1,143 @@
+import time
+import logging
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+
+from ocs_ci.utility.utils import run_cmd, get_kubeadmin_password
+from ocs_ci.ocs.ui_selenium.const_xpath import xpath_pvc_page
+
+logger = logging.getLogger(__name__)
+
+
+class UserInterfaceSelenium(object):
+    """
+    User Interface Selenium
+
+    """
+
+    def __init__(self):
+        self.driver = None
+
+    def ui_login(self):
+        """
+        Login to Openshift Console
+
+        """
+        console_url = run_cmd(
+            "oc get consoles.config.openshift.io cluster -o"
+            "jsonpath='{.status.consoleURL}'"
+        )
+        password = get_kubeadmin_password()
+        password = password.rstrip()
+        self.driver = webdriver.Chrome()
+        self.driver.get(console_url)
+        time.sleep(30)
+        self.driver.find_element_by_id("inputUsername").send_keys("kubeadmin")
+        self.driver.find_element_by_id("inputPassword").send_keys(password)
+        self.click_xpath("/html/body/div/div/main/div/form/div[4]/button")
+        time.sleep(30)
+        self.driver.get(console_url + "/dashboards")
+        time.sleep(20)
+
+    def create_pvc_ui(self, sc_type, pvc_name, access_mode, pvc_size):
+        """
+        Create PVC via UI.
+
+        sc_type (str): storage class type
+        pvc_name (str): the name of pvc
+        access_mode (str): access mode
+        pvc_size (str): the size of pvc (GB)
+
+        """
+        logger.info("Click on Storage Tab")
+        self.click_xpath(xpath_pvc_page["Storage Tab"])
```

Comment:
Please add here a check if Storage option is already expanded. If it is, don't click it.